child_update_picture_date/wizard/__init__.py

```diff
@@ -0,0 +1,12 @@
+# -*- encoding: utf-8 -*-

```

Comment:
Because we launched the update on friday night (the process take one day!), and the cron is convenient for executing background tasks.